src/index.html

```diff
@@ -2,11 +2,394 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        alt="Moyo"
+        class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              class="nav__link "
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="cards">
+        <div class="product-card" data-qa="card">
+          <div class="product-logo"></div>
+
+          <h1 class="product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+          <div class="product-code">Product code: 195434</div>
```

Comment:
Consider using semantic tags for text
```suggestion
          <p class="product-code">Product code: 195434</p>
```

Reply:
If I use the p tag - extra space is created around it and breaks everything

Reply:
just reset default margin for this tag